cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id="viewing-pipeline-logs-using-the-openshift-logging-operator"]
+= Viewing pipeline logs using the OpenShift Logging Operator
+
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: viewing-pipeline-logs-using-the-openshift-logging-operator
```

Comment:
```suggestion
[id="viewing-pipeline-logs-using-the-openshift-logging-operator"]
= Viewing pipeline logs using the OpenShift Logging Operator
include::modules/common-attributes.adoc[]
include::modules/pipelines-document-attributes.adoc[]
:context: viewing-pipeline-logs-using-the-openshift-logging-operator
```

Please carefully review the OCP guidelines going forward: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata
I think this spacing is causing rendering issues.